.github/workflows/tests.yaml

```diff
@@ -27,8 +27,11 @@ jobs:
           sudo snap install --no-wait --classic pyright
           sudo snap install --no-wait shellcheck
           echo "::endgroup::"
+          echo "::group::apt-get"
+          sudo apt-get install -y libapt-pkg-dev
+          echo "::endgroup::"
           echo "::group::pip install"
-          python -m pip install 'tox>=4' tox-gh
```

Comment:
Is this because the linters only run on python 3.10?  Can this be removed in starbase too?

Reply:
It's because  I moved the `tox-gh` dependency into `tox.ini` where renovate can control it too. I want to test it here but presuming it all works I'll move it into starbase.